Index: app/src/main/res/layout/content_scrolling.xml
===================================================================
--- app/src/main/res/layout/content_scrolling.xml	(revision c2c6a0373bddc1a4a140888f3a06c66462b7ee13)
+++ app/src/main/res/layout/content_scrolling.xml	(revision c2c6a0373bddc1a4a140888f3a06c66462b7ee13)
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    app:layout_behavior="@string/appbar_scrolling_view_behavior"
-    tools:context=".SettingsOfChart"
-    tools:showIn="@layout/activity_settings_of_chart">
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/text_margin"
-        android:text="@string/large_text" />
-
-</androidx.core.widget.NestedScrollView>
\ No newline at end of file
Index: app/src/main/java/com/example/myapplication/FirebaseFirestoreConnection.java
===================================================================
--- app/src/main/java/com/example/myapplication/FirebaseFirestoreConnection.java	(revision c2c6a0373bddc1a4a140888f3a06c66462b7ee13)
+++ app/src/main/java/com/example/myapplication/FirebaseFirestoreConnection.java	(revision c2c6a0373bddc1a4a140888f3a06c66462b7ee13)
@@ -1,29 +0,0 @@
-/*package com.example.myapplication;
-
-import com.google.auth.oauth2.GoogleCredentials;
-import com.google.firebase.FirebaseApp;
-import com.google.firebase.FirebaseOptions;
-
-import java.io.FileInputStream;
-
-import javax.annotation.PostConstruct;
-
-public class FirebaseFirestoreConnection {
-
-    @PostConstruct
-    public void initialize(){
-        try {
-            FileInputStream serviceAccount =
-                    new FileInputStream("./app/FirebaseService.json");
-
-            FirebaseOptions options = new FirebaseOptions.Builder()
-                    .setCredentials(GoogleCredentials.fromStream(serviceAccount))
-                    .setDatabaseUrl("https://moneyassistant-1604315426885.firebaseio.com")
-                    .build();
-
-            FirebaseApp.initializeApp(options);
-        } catch (Exception e){
-            e.printStackTrace();
-        }
-    }
-}*/
